sample/db/samples/tax_rates.rb

```diff
@@ -3,7 +3,10 @@
 tax_rate = Spree::TaxRate.create(
   name: "North America",
   zone: north_america,
-  amount: 0.05,
-  tax_category: clothing)
+  amount: 0.05)
```

Comment:
Nitpick: I know it was like this before, but could we move the closing paren to the next line? A bit hard to read as-is.